Resources/Prototypes/Parkstation/Entities/Mobs/Player/shadowkin.yml

```diff
@@ -0,0 +1,144 @@
+﻿- type: entity
+  save: false
+  parent: BaseMobHuman
+  id: MobShadowkin
+  name: Urist McFloof
+  components:
+    - type: HumanoidAppearance
+      species: Shadowkin
+    - type: Hunger
+    - type: Thirst
+    - type: Icon
+      sprite: Parkstation/Mobs/Species/shadowkin.rsi
+      state: full
+    - type: Body
+      prototype: Shadowkin
+      requiredLegs: 2
+    # - type: DiseaseCarrier
+      # diseaseResist: 0.1
+    - type: Flammable
+      damage:
+        types:
+          Heat: 1.5 # burn more
+    - type: MobState
+    - type: MobThresholds
+      thresholds: # Weak
+        0: Alive
+        80: Critical
+        160: Dead
```

Comment:
Should probably change Dead to 180, there's some issues with medical balance if the Crit-Dead range is not equal to 100.